src/getRandomIntInclusive.js

```diff
@@ -0,0 +1,8 @@
+// https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+function getRandomIntInclusive(min, max) {
+  min = Math.ceil(min);
+  max = Math.floor(max);
```

Comment:
재할당해서 사용하는 경우 의도하지 않은 부수 효과가 생길 수 있을 것 같아요.     
이름을 굳이 다시 붙여야 하는 점은 좀 별로지만..ㅋㅋ 새로 할당해서 사용하면 어떨까요?    
```suggestion
  const ceilMin = Math.ceil(min);
  const floorMax = Math.floor(max);
```

Reply:
섀도잉이라고 부르죠. 🙂 위의 링크에서 코드를 그대로 복사하였습니다.